mmv1/third_party/terraform/services/datastream/resource_datastream_connection_profile_test.go

```diff
@@ -72,6 +75,28 @@ func TestAccDatastreamConnectionProfile_update(t *testing.T) {
 				// Disable prevent_destroy
 				Config: testAccDatastreamConnectionProfile_mySQLUpdate(context, false, random_pass_2),
 			},
+			{
+				Config: testAccDatastreamConnectionProfile_SSHKey_Update(context, true, random_key_1),
```

Comment:
This is a little strange, because destroy gets run after all steps are complete, and with the same instance name is being used, so I think it will try to update from mysql to postgres. I would create a new test if you need to use a postgres instance.